hopsworks-IT/src/test/ruby/spec/featurestore_spec.rb

```diff
@@ -166,10 +167,104 @@
           expect(parsed_json["bucket"] == "testbucket").to be true
         end
 
+        it "should be able to add s3 connector to the featurestore with encryption algorithm but no key" do
+          project = get_project
+          encryption_algorithm = "AES-256"
+          encryption_key = "Test"
```

Comment:
You shouldn't be required to pass an encryption key if the if the algorithm is `AES-256`